sequencingSourceFileDetails.md

```diff
@@ -0,0 +1,35 @@
+CCH163.SG	SAMEA7050467	PRJEB39316	ERR4332645	15	ERS4811097	2021-04-12	2020-07-10	Illumina NovaSeq 6000	SINGLE	GENOMIC	ILLUMINA	CCH163_L1	WGS	fasp.sra.ebi.ac.uk:/vol1/fastq/ERR433/005/ERR4332645/ERR4332645.fastq.gz	86634518	513794058fdf06e160cad3579241fc19	ftp.sra.ebi.ac.uk/vol1/fastq/ERR433/005/ERR4332645/ERR4332645.fastq.gz	2428370	ftp.sra.ebi.ac.uk/vol1/run/ERR433/ERR4332645/CCH163-b1e1l1p1_CGTAATC-ATGGTCT_L001_ARmerged.200115_A00621_0169_BHYLNNDSXX.all.hs37d5.cons.90perc.bam
+CCH285.SG	SAMEA7050468	PRJEB39316	ERR4332646	16	ERS4811098	2021-04-12	2020-07-10	Illumina NovaSeq 6000	SINGLE	GENOMIC	ILLUMINA	CCH285_L1	WGS	fasp.sra.ebi.ac.uk:/vol1/fastq/ERR433/006/ERR4332646/ERR4332646.fastq.gz	184093891	d3af0bb0802d2da7d68df3cabbeb4841	ftp.sra.ebi.ac.uk/vol1/fastq/ERR433/006/ERR4332646/ERR4332646.fastq.gz	4879361	ftp.sra.ebi.ac.uk/vol1/run/ERR433/ERR4332646/CCH285-b1e1l1p1_GTTATAC-ACTTCAA_L001_ARmerged.200115_A00621_0169_BHYLNNDSXX.all.hs37d5.cons.90perc.bam
+CCH289.SG	SAMEA7050469	PRJEB39316	ERR4332647	17	ERS4811099	2021-04-12	2020-07-10	Illumina NovaSeq 6000	SINGLE	GENOMIC	ILLUMINA	CCH289_L1	WGS	fasp.sra.ebi.ac.uk:/vol1/fastq/ERR433/007/ERR4332647/ERR4332647.fastq.gz	179891674	a9d1316845f7c8dd6a3852a82cfc4c20	ftp.sra.ebi.ac.uk/vol1/fastq/ERR433/007/ERR4332647/ERR4332647.fastq.gz	5013829	ftp.sra.ebi.ac.uk/vol1/run/ERR433/ERR4332647/CCH289-b1e1l1p1_CTTGCGG-CTCATGG_L001_ARmerged.200115_A00621_0169_BHYLNNDSXX.all.hs37d5.cons.90perc.bam
+CCH290.SG	SAMEA7050470	PRJEB39316	ERR4332648	18	ERS4811100	2021-04-12	2020-07-10	Illumina NovaSeq 6000	SINGLE	GENOMIC	ILLUMINA	CCH290_L1	WGS	fasp.sra.ebi.ac.uk:/vol1/fastq/ERR433/008/ERR4332648/ERR4332648.fastq.gz	30401831	352d76ddb75edee7e30ba0800940b126	ftp.sra.ebi.ac.uk/vol1/fastq/ERR433/008/ERR4332648/ERR4332648.fastq.gz	765274	ftp.sra.ebi.ac.uk/vol1/run/ERR433/ERR4332648/CCH290-b1e1l1p1_AATCTTC-TGATAGT_L001_ARmerged.200115_A00621_0169_BHYLNNDSXX.all.hs37d5.cons.90perc.bam
+CCH294.SG	SAMEA7050471	PRJEB39316	ERR4332649	19	ERS4811101	2021-04-12	2020-07-10	Illumina NovaSeq 6000	SINGLE	GENOMIC	ILLUMINA	CCH294_L1	WGS	fasp.sra.ebi.ac.uk:/vol1/fastq/ERR433/009/ERR4332649/ERR4332649.fastq.gz	610643404	6a5689763b239ff60c24bc5313892c6e	ftp.sra.ebi.ac.uk/vol1/fastq/ERR433/009/ERR4332649/ERR4332649.fastq.gz	15356366	ftp.sra.ebi.ac.uk/vol1/run/ERR433/ERR4332649/CCH294-b1e1l1p1_ACCAACG-ATTAAGG_L001_ARmerged.200115_A00621_0169_BHYLNNDSXX.all.hs37d5.cons.90perc.bam
+CCH311.SG	SAMEA7050472	PRJEB39316	ERR4332650	20	ERS4811102	2021-04-12	2020-07-10	Illumina NovaSeq 6000	SINGLE	GENOMIC	ILLUMINA	CCH311_L1	WGS	fasp.sra.ebi.ac.uk:/vol1/fastq/ERR433/000/ERR4332650/ERR4332650.fastq.gz	308098723	b67b26467df5117d6269793a9a63349e	ftp.sra.ebi.ac.uk/vol1/fastq/ERR433/000/ERR4332650/ERR4332650.fastq.gz	7539845	ftp.sra.ebi.ac.uk/vol1/run/ERR433/ERR4332650/CCH311-b1e1l1p1_AACCATG-CTACTCT_L001_ARmerged.200115_A00621_0169_BHYLNNDSXX.all.hs37d5.cons.90perc.bam
```

Comment:
Maybe 5 rows would be enough for the example.

Reply:
Done!